releasenotes/notes/rtloader-python-lazy-loading-f16eed252c3176e7.yaml

```
@@ -0,0 +1,13 @@
+# Each section from every release note are combined when the
+# CHANGELOG.rst is rendered. So the text needs to be worded so that
+# it does not depend on any information only available in another
+# section. This may mean repeating some details, but each section
+# must be readable independently of the other.
+#
+# Each section note must be formatted as reStructuredText.
+---
+enhancements:
+  - |
+    The Python runtime is now loaded when the first check is loaded, rather than when the agent starts.
+    This will eventually allow not loading Python when no Python check is enabled.
```

```suggestion
    This will eventually prevent the loading of Python when no Python check is enabled.
```

---

I think this is clearer, unless I'm misunderstand what you're saying here.

---

@michaelcretzman I'm not sure that's quite equivalent to what I mean 🤔 
It's not that we are accidentally loading Python, we're doing it on purpose, but we'd like to be able not to do it

What do you think of the following ?
```suggestion
    This will eventually avoid having to load Python when no Python check is enabled.
```

---

that's clearer. thanks!